curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-dom-manipulation-by-building-a-role-playing-game/62a7c011eef9fb2084b966db.md

```diff
@@ -1,8 +1,8 @@
 ---
 id: 62a7c011eef9fb2084b966db
-title: Step 47
+title: Step 41
 challengeType: 0
-dashedName: step-47
+dashedName: step-41
```

Comment:
whats the purpose of moving the function above another function ? maybe add a quick explainer or remove this mini step ?

Reply:
We could add a quick note